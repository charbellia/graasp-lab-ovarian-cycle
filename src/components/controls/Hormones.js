import React from 'react';

export const Hormones = ({ d3 }) => (
  <div className="Hormone-Container">
    <div className="hormones-container">
      <p>
        <span className="fsh-puces">.</span>
        <span className="fsh-container">FSH</span>
        <span className="fsh-state">+</span>
      </p>
      <p>
        <span className="lh-puces">.</span>
        <span className="lh-container">LH</span>
        <span className="lh-state">+</span>
      </p>
      <p>
        <span className="oestro-puces">.</span>
        <span className="oestro-container">OESTROGÈNES</span>
        <span className="oestro-state">-</span>
      </p>
      <p>
        <span className="progest-puces">.</span>
        <span className="progest-container">PROGESTÉRONES</span>
        <span className="progest-state">-</span>
      </p>
    </div>
  </div>
);

export default (Hormones);
