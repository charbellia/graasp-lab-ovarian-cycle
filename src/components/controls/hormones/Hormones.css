.hormone-container {
  width: 32rem;
  margin: auto;
}
.hormones-table {
  border: 4px solid #DDD;
  padding: 1rem 4rem;
}
.lh-name,
.fsh-name,
.oestro-name,
.progest-name {
  background: #fff;
  font-size: 2rem;
  width: 14rem;
  margin: 5px auto;
  text-align: center;
}
.fsh-state,
.lh-state,
.oestro-state,
.progest-state {
  float: right;
  margin-top: -42px;
  padding: 3px 10px;
  background: #fff;
 }
 .fsh-indicator,
 .lh-indicator,
 .oestro-indicator,
 .progest-indicator {
   padding: 10px;
   width: 30px;
   border-radius: 50%;
   height: 30px;
   float: left;
   margin-top: 5px;
 }
.fsh-indicator {
   background-color: #ff7a00;
 }
.lh-indicator {
   background-color: #12a3c1;
 }
.oestro-indicator {
   background-color: #3bc71f;
 }
.progest-indicator {
   background-color: #9C27B0;
 }
 .description-container {
   margin-top: 2rem;
   padding: 1rem;
 }
 .ovulation-description,
 .preovulation-description,
 .postovulation-description {
   font-size: 2.2rem;
   font-weight: 300;
 }
 .ovulation-description {
   color: #ff7a00;
 }
 .postovulation-description {
   color: #9C27B0;
 }
 .preovulation-description {
   color: #12a3c1;
 }
 .active-preovulation {
   background-color: #12a3c1 !important;
   border-color: #12a3c1 !important;
 }
 .active-ovulation {
   background-color: #ff7a00 !important;
   border-color: #ff7a00 !important;
 }
 .active-postovulation {
   background-color: #9C27B0 !important;
   border-color: #9C27B0 !important;
 }

.animate-text,
.ovulation-description,
.preovulation-description,
.postovulation-description {
   animation-name: fade;
   animation-fill-mode: both;
   animation-duration: 1s;
   animation-direction: alternate;
 }

@keyframes fade {
  0%,50% {
    opacity: 0;
  }
  100%{
    opacity: 1;
  }
}
