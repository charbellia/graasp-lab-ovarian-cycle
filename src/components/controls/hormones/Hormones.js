import React from 'react';
import PropTypes from 'prop-types';
import './Hormones.css';

export const Hormones = ({
  ovulation,
  postOvulation,
  preOvulation,
  secreteLhFsh,
  secreteProgest,
  secreteOestro,
}) => (
  <div className="Hormone-Container">
    <div className="hormones-container">
      <div className="hormones-table">
        <div className="lh-hormones">
          <span className={`${secreteLhFsh ? 'animate-lh-hormones' : ''} lh-indicator`} />
          <div className={`${secreteLhFsh ? 'animate-lh-hormones' : ''} lh-name`}>LH</div>
          <span className={`${secreteLhFsh ? 'animate-lh-hormones' : ''} lh-state`}>
            {secreteLhFsh ? '+' : '-'}
          </span>
        </div>
        <div className="fsh-hormones">
          <span className={`${secreteLhFsh ? 'animate-fsh-hormones' : ''} fsh-indicator`} />
          <div className={`${secreteLhFsh ? 'animate-fsh-hormones' : ''} fsh-name`}>FSH</div>
          <span className={`${secreteLhFsh ? 'animate-fsh-hormones' : ''} fsh-state`}>
            {secreteLhFsh ? '+' : '-'}
          </span>
        </div>
        <div className="oestro-hormones">
          <span className={`${secreteOestro ? 'animate-oestro-hormones' : ''} oestro-indicator`} />
          <div className={`${secreteOestro ? 'animate-oestro-hormones' : ''} oestro-name`}>Oestrogens</div>
          <span className={`${secreteOestro ? 'animate-oestro-hormones' : ''} oestro-state`}>
            {secreteOestro ? '+' : '-'}
          </span>
        </div>
        <div className="progest-hormones">
          <span className={`${secreteProgest ? 'animate-progest-hormones' : ''} progest-indicator`} />
          <div className={`${secreteProgest ? 'animate-progest-hormones' : ''} progest-name`}>Progesterones</div>
          <span className={`${secreteProgest ? 'animate-progest-hormones' : ''} progest-state`}>
            {secreteProgest ? '+' : '-'}
          </span>
        </div>
      </div>
      <div className="description-container">
        <h2 className={`${preOvulation ? 'animate-text' : ''}`}>
          {preOvulation ? 'Pre-Ovulation' : ''}
        </h2>
        <h2 className={`${ovulation ? 'animate-text' : ''}`}>
          {ovulation ? 'Ovulation' : ''}
        </h2>
        <h2 className={`${postOvulation ? 'animate-text' : ''}`}>
          {postOvulation ? 'Post-Ovulation' : ''}
        </h2>

        <p className="Phase-description">
          {postOvulation ? 'This stage is the moment when the LH and FSH are in very low quatity.' : ''}
          {preOvulation ? 'This stage is the moment when the LH and FSH are in normal quatity.' : ''}
          {ovulation ? 'This stage is the moment when LH and FSH as well as estrogens are in great quantity.' : ''}
        </p>
        <p className="Phase-description">
          {postOvulation ? 'This is the period after ovulation.' : ''}
          {preOvulation ? 'This is the period before ovulation' : ''}
          {ovulation ? 'This is the ovulation phase.' : ''}
        </p>
      </div>
    </div>
  </div>
);

Hormones.propTypes = {
  ovulation: PropTypes.bool.isRequired,
  postOvulation: PropTypes.bool.isRequired,
  preOvulation: PropTypes.bool.isRequired,
  secreteLhFsh: PropTypes.bool.isRequired,
  secreteOestro: PropTypes.bool.isRequired,
  secreteProgest: PropTypes.bool.isRequired,
};

export default (Hormones);
