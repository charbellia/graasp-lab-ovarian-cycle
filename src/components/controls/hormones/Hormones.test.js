import React from 'react';
import { shallow } from 'enzyme';
import { Hormones } from './Hormones';

describe('<Hormones />', () => {
  const props = {
    ovulation: false,
    postOvulation: false,
    preOvulation: false,
    secreteLhFsh: false,
    secreteProgest: false,
    secreteOestro: false,
    t: jest.fn(),
    classes: {},
  };

  const component = shallow(<Hormones {...props} />);

  it('renders correctly', () => {
    expect(component).toMatchSnapshot();
  });
});
