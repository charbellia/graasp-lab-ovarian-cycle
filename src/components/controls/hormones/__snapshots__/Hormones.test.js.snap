// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hormones /> renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Hormones
    classes={Object {}}
    ovulation={false}
    postOvulation={false}
    preOvulation={false}
    secreteLhFsh={false}
    secreteOestro={false}
    secreteProgest={false}
    t={
      [MockFunction] {
        "calls": Array [
          Array [
            "LH",
          ],
          Array [
            "FSH",
          ],
          Array [
            "Oestrogens",
          ],
          Array [
            "Progesterones",
          ],
        ],
        "results": Array [
          Object {
            "isThrow": false,
            "value": undefined,
          },
          Object {
            "isThrow": false,
            "value": undefined,
          },
          Object {
            "isThrow": false,
            "value": undefined,
          },
          Object {
            "isThrow": false,
            "value": undefined,
          },
        ],
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <WithStyles(Grid)
        container={true}
        spacing={16}
      >
        <WithStyles(Grid)
          item={true}
          md={6}
          xs={12}
        >
          <div>
            <WithStyles(List)
              dense={false}
            >
              <WithStyles(ListItem)>
                <WithStyles(ListItemIcon)>
                  <pure(FiberManualRecordIcon) />
                </WithStyles(ListItemIcon)>
                <WithStyles(ListItemText)
                  className=" lh-name"
                  secondary="undefined"
                />
              </WithStyles(ListItem)>
              <WithStyles(ListItem)>
                <WithStyles(ListItemIcon)>
                  <pure(FiberManualRecordIcon) />
                </WithStyles(ListItemIcon)>
                <WithStyles(ListItemText)
                  className=" fsh-name"
                  secondary="undefined"
                />
              </WithStyles(ListItem)>
              <WithStyles(ListItem)>
                <WithStyles(ListItemIcon)>
                  <pure(FiberManualRecordIcon) />
                </WithStyles(ListItemIcon)>
                <WithStyles(ListItemText)
                  className=" oestro-name"
                  secondary="undefined"
                />
              </WithStyles(ListItem)>
              <WithStyles(ListItem)>
                <WithStyles(ListItemIcon)>
                  <pure(FiberManualRecordIcon) />
                </WithStyles(ListItemIcon)>
                <WithStyles(ListItemText)
                  className=" progest-name"
                  secondary="undefined"
                />
              </WithStyles(ListItem)>
            </WithStyles(List)>
          </div>
        </WithStyles(Grid)>
      </WithStyles(Grid)>,
      "className": undefined,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <WithStyles(Grid)
          item={true}
          md={6}
          xs={12}
        >
          <div>
            <WithStyles(List)
              dense={false}
            >
              <WithStyles(ListItem)>
                <WithStyles(ListItemIcon)>
                  <pure(FiberManualRecordIcon) />
                </WithStyles(ListItemIcon)>
                <WithStyles(ListItemText)
                  className=" lh-name"
                  secondary="undefined"
                />
              </WithStyles(ListItem)>
              <WithStyles(ListItem)>
                <WithStyles(ListItemIcon)>
                  <pure(FiberManualRecordIcon) />
                </WithStyles(ListItemIcon)>
                <WithStyles(ListItemText)
                  className=" fsh-name"
                  secondary="undefined"
                />
              </WithStyles(ListItem)>
              <WithStyles(ListItem)>
                <WithStyles(ListItemIcon)>
                  <pure(FiberManualRecordIcon) />
                </WithStyles(ListItemIcon)>
                <WithStyles(ListItemText)
                  className=" oestro-name"
                  secondary="undefined"
                />
              </WithStyles(ListItem)>
              <WithStyles(ListItem)>
                <WithStyles(ListItemIcon)>
                  <pure(FiberManualRecordIcon) />
                </WithStyles(ListItemIcon)>
                <WithStyles(ListItemText)
                  className=" progest-name"
                  secondary="undefined"
                />
              </WithStyles(ListItem)>
            </WithStyles(List)>
          </div>
        </WithStyles(Grid)>,
        "container": true,
        "spacing": 16,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <div>
            <WithStyles(List)
              dense={false}
            >
              <WithStyles(ListItem)>
                <WithStyles(ListItemIcon)>
                  <pure(FiberManualRecordIcon) />
                </WithStyles(ListItemIcon)>
                <WithStyles(ListItemText)
                  className=" lh-name"
                  secondary="undefined"
                />
              </WithStyles(ListItem)>
              <WithStyles(ListItem)>
                <WithStyles(ListItemIcon)>
                  <pure(FiberManualRecordIcon) />
                </WithStyles(ListItemIcon)>
                <WithStyles(ListItemText)
                  className=" fsh-name"
                  secondary="undefined"
                />
              </WithStyles(ListItem)>
              <WithStyles(ListItem)>
                <WithStyles(ListItemIcon)>
                  <pure(FiberManualRecordIcon) />
                </WithStyles(ListItemIcon)>
                <WithStyles(ListItemText)
                  className=" oestro-name"
                  secondary="undefined"
                />
              </WithStyles(ListItem)>
              <WithStyles(ListItem)>
                <WithStyles(ListItemIcon)>
                  <pure(FiberManualRecordIcon) />
                </WithStyles(ListItemIcon)>
                <WithStyles(ListItemText)
                  className=" progest-name"
                  secondary="undefined"
                />
              </WithStyles(ListItem)>
            </WithStyles(List)>
          </div>,
          "item": true,
          "md": 6,
          "xs": 12,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <WithStyles(List)
              dense={false}
            >
              <WithStyles(ListItem)>
                <WithStyles(ListItemIcon)>
                  <pure(FiberManualRecordIcon) />
                </WithStyles(ListItemIcon)>
                <WithStyles(ListItemText)
                  className=" lh-name"
                  secondary="undefined"
                />
              </WithStyles(ListItem)>
              <WithStyles(ListItem)>
                <WithStyles(ListItemIcon)>
                  <pure(FiberManualRecordIcon) />
                </WithStyles(ListItemIcon)>
                <WithStyles(ListItemText)
                  className=" fsh-name"
                  secondary="undefined"
                />
              </WithStyles(ListItem)>
              <WithStyles(ListItem)>
                <WithStyles(ListItemIcon)>
                  <pure(FiberManualRecordIcon) />
                </WithStyles(ListItemIcon)>
                <WithStyles(ListItemText)
                  className=" oestro-name"
                  secondary="undefined"
                />
              </WithStyles(ListItem)>
              <WithStyles(ListItem)>
                <WithStyles(ListItemIcon)>
                  <pure(FiberManualRecordIcon) />
                </WithStyles(ListItemIcon)>
                <WithStyles(ListItemText)
                  className=" progest-name"
                  secondary="undefined"
                />
              </WithStyles(ListItem)>
            </WithStyles(List)>,
            "className": undefined,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <WithStyles(ListItem)>
                  <WithStyles(ListItemIcon)>
                    <pure(FiberManualRecordIcon) />
                  </WithStyles(ListItemIcon)>
                  <WithStyles(ListItemText)
                    className=" lh-name"
                    secondary="undefined"
                  />
                </WithStyles(ListItem)>,
                <WithStyles(ListItem)>
                  <WithStyles(ListItemIcon)>
                    <pure(FiberManualRecordIcon) />
                  </WithStyles(ListItemIcon)>
                  <WithStyles(ListItemText)
                    className=" fsh-name"
                    secondary="undefined"
                  />
                </WithStyles(ListItem)>,
                <WithStyles(ListItem)>
                  <WithStyles(ListItemIcon)>
                    <pure(FiberManualRecordIcon) />
                  </WithStyles(ListItemIcon)>
                  <WithStyles(ListItemText)
                    className=" oestro-name"
                    secondary="undefined"
                  />
                </WithStyles(ListItem)>,
                <WithStyles(ListItem)>
                  <WithStyles(ListItemIcon)>
                    <pure(FiberManualRecordIcon) />
                  </WithStyles(ListItemIcon)>
                  <WithStyles(ListItemText)
                    className=" progest-name"
                    secondary="undefined"
                  />
                </WithStyles(ListItem)>,
              ],
              "dense": false,
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": Array [
                    <WithStyles(ListItemIcon)>
                      <pure(FiberManualRecordIcon) />
                    </WithStyles(ListItemIcon)>,
                    <WithStyles(ListItemText)
                      className=" lh-name"
                      secondary="undefined"
                    />,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": <pure(FiberManualRecordIcon) />,
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "class",
                      "props": Object {
                        "className": undefined,
                      },
                      "ref": null,
                      "rendered": null,
                      "type": [Function],
                    },
                    "type": [Function],
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "className": " lh-name",
                      "secondary": "undefined",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                ],
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": Array [
                    <WithStyles(ListItemIcon)>
                      <pure(FiberManualRecordIcon) />
                    </WithStyles(ListItemIcon)>,
                    <WithStyles(ListItemText)
                      className=" fsh-name"
                      secondary="undefined"
                    />,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": <pure(FiberManualRecordIcon) />,
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "class",
                      "props": Object {
                        "className": undefined,
                      },
                      "ref": null,
                      "rendered": null,
                      "type": [Function],
                    },
                    "type": [Function],
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "className": " fsh-name",
                      "secondary": "undefined",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                ],
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": Array [
                    <WithStyles(ListItemIcon)>
                      <pure(FiberManualRecordIcon) />
                    </WithStyles(ListItemIcon)>,
                    <WithStyles(ListItemText)
                      className=" oestro-name"
                      secondary="undefined"
                    />,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": <pure(FiberManualRecordIcon) />,
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "class",
                      "props": Object {
                        "className": undefined,
                      },
                      "ref": null,
                      "rendered": null,
                      "type": [Function],
                    },
                    "type": [Function],
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "className": " oestro-name",
                      "secondary": "undefined",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                ],
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": Array [
                    <WithStyles(ListItemIcon)>
                      <pure(FiberManualRecordIcon) />
                    </WithStyles(ListItemIcon)>,
                    <WithStyles(ListItemText)
                      className=" progest-name"
                      secondary="undefined"
                    />,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": <pure(FiberManualRecordIcon) />,
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "class",
                      "props": Object {
                        "className": undefined,
                      },
                      "ref": null,
                      "rendered": null,
                      "type": [Function],
                    },
                    "type": [Function],
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "className": " progest-name",
                      "secondary": "undefined",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                ],
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          "type": "div",
        },
        "type": [Function],
      },
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <WithStyles(Grid)
          container={true}
          spacing={16}
        >
          <WithStyles(Grid)
            item={true}
            md={6}
            xs={12}
          >
            <div>
              <WithStyles(List)
                dense={false}
              >
                <WithStyles(ListItem)>
                  <WithStyles(ListItemIcon)>
                    <pure(FiberManualRecordIcon) />
                  </WithStyles(ListItemIcon)>
                  <WithStyles(ListItemText)
                    className=" lh-name"
                    secondary="undefined"
                  />
                </WithStyles(ListItem)>
                <WithStyles(ListItem)>
                  <WithStyles(ListItemIcon)>
                    <pure(FiberManualRecordIcon) />
                  </WithStyles(ListItemIcon)>
                  <WithStyles(ListItemText)
                    className=" fsh-name"
                    secondary="undefined"
                  />
                </WithStyles(ListItem)>
                <WithStyles(ListItem)>
                  <WithStyles(ListItemIcon)>
                    <pure(FiberManualRecordIcon) />
                  </WithStyles(ListItemIcon)>
                  <WithStyles(ListItemText)
                    className=" oestro-name"
                    secondary="undefined"
                  />
                </WithStyles(ListItem)>
                <WithStyles(ListItem)>
                  <WithStyles(ListItemIcon)>
                    <pure(FiberManualRecordIcon) />
                  </WithStyles(ListItemIcon)>
                  <WithStyles(ListItemText)
                    className=" progest-name"
                    secondary="undefined"
                  />
                </WithStyles(ListItem)>
              </WithStyles(List)>
            </div>
          </WithStyles(Grid)>
        </WithStyles(Grid)>,
        "className": undefined,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <WithStyles(Grid)
            item={true}
            md={6}
            xs={12}
          >
            <div>
              <WithStyles(List)
                dense={false}
              >
                <WithStyles(ListItem)>
                  <WithStyles(ListItemIcon)>
                    <pure(FiberManualRecordIcon) />
                  </WithStyles(ListItemIcon)>
                  <WithStyles(ListItemText)
                    className=" lh-name"
                    secondary="undefined"
                  />
                </WithStyles(ListItem)>
                <WithStyles(ListItem)>
                  <WithStyles(ListItemIcon)>
                    <pure(FiberManualRecordIcon) />
                  </WithStyles(ListItemIcon)>
                  <WithStyles(ListItemText)
                    className=" fsh-name"
                    secondary="undefined"
                  />
                </WithStyles(ListItem)>
                <WithStyles(ListItem)>
                  <WithStyles(ListItemIcon)>
                    <pure(FiberManualRecordIcon) />
                  </WithStyles(ListItemIcon)>
                  <WithStyles(ListItemText)
                    className=" oestro-name"
                    secondary="undefined"
                  />
                </WithStyles(ListItem)>
                <WithStyles(ListItem)>
                  <WithStyles(ListItemIcon)>
                    <pure(FiberManualRecordIcon) />
                  </WithStyles(ListItemIcon)>
                  <WithStyles(ListItemText)
                    className=" progest-name"
                    secondary="undefined"
                  />
                </WithStyles(ListItem)>
              </WithStyles(List)>
            </div>
          </WithStyles(Grid)>,
          "container": true,
          "spacing": 16,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <div>
              <WithStyles(List)
                dense={false}
              >
                <WithStyles(ListItem)>
                  <WithStyles(ListItemIcon)>
                    <pure(FiberManualRecordIcon) />
                  </WithStyles(ListItemIcon)>
                  <WithStyles(ListItemText)
                    className=" lh-name"
                    secondary="undefined"
                  />
                </WithStyles(ListItem)>
                <WithStyles(ListItem)>
                  <WithStyles(ListItemIcon)>
                    <pure(FiberManualRecordIcon) />
                  </WithStyles(ListItemIcon)>
                  <WithStyles(ListItemText)
                    className=" fsh-name"
                    secondary="undefined"
                  />
                </WithStyles(ListItem)>
                <WithStyles(ListItem)>
                  <WithStyles(ListItemIcon)>
                    <pure(FiberManualRecordIcon) />
                  </WithStyles(ListItemIcon)>
                  <WithStyles(ListItemText)
                    className=" oestro-name"
                    secondary="undefined"
                  />
                </WithStyles(ListItem)>
                <WithStyles(ListItem)>
                  <WithStyles(ListItemIcon)>
                    <pure(FiberManualRecordIcon) />
                  </WithStyles(ListItemIcon)>
                  <WithStyles(ListItemText)
                    className=" progest-name"
                    secondary="undefined"
                  />
                </WithStyles(ListItem)>
              </WithStyles(List)>
            </div>,
            "item": true,
            "md": 6,
            "xs": 12,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <WithStyles(List)
                dense={false}
              >
                <WithStyles(ListItem)>
                  <WithStyles(ListItemIcon)>
                    <pure(FiberManualRecordIcon) />
                  </WithStyles(ListItemIcon)>
                  <WithStyles(ListItemText)
                    className=" lh-name"
                    secondary="undefined"
                  />
                </WithStyles(ListItem)>
                <WithStyles(ListItem)>
                  <WithStyles(ListItemIcon)>
                    <pure(FiberManualRecordIcon) />
                  </WithStyles(ListItemIcon)>
                  <WithStyles(ListItemText)
                    className=" fsh-name"
                    secondary="undefined"
                  />
                </WithStyles(ListItem)>
                <WithStyles(ListItem)>
                  <WithStyles(ListItemIcon)>
                    <pure(FiberManualRecordIcon) />
                  </WithStyles(ListItemIcon)>
                  <WithStyles(ListItemText)
                    className=" oestro-name"
                    secondary="undefined"
                  />
                </WithStyles(ListItem)>
                <WithStyles(ListItem)>
                  <WithStyles(ListItemIcon)>
                    <pure(FiberManualRecordIcon) />
                  </WithStyles(ListItemIcon)>
                  <WithStyles(ListItemText)
                    className=" progest-name"
                    secondary="undefined"
                  />
                </WithStyles(ListItem)>
              </WithStyles(List)>,
              "className": undefined,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": Array [
                  <WithStyles(ListItem)>
                    <WithStyles(ListItemIcon)>
                      <pure(FiberManualRecordIcon) />
                    </WithStyles(ListItemIcon)>
                    <WithStyles(ListItemText)
                      className=" lh-name"
                      secondary="undefined"
                    />
                  </WithStyles(ListItem)>,
                  <WithStyles(ListItem)>
                    <WithStyles(ListItemIcon)>
                      <pure(FiberManualRecordIcon) />
                    </WithStyles(ListItemIcon)>
                    <WithStyles(ListItemText)
                      className=" fsh-name"
                      secondary="undefined"
                    />
                  </WithStyles(ListItem)>,
                  <WithStyles(ListItem)>
                    <WithStyles(ListItemIcon)>
                      <pure(FiberManualRecordIcon) />
                    </WithStyles(ListItemIcon)>
                    <WithStyles(ListItemText)
                      className=" oestro-name"
                      secondary="undefined"
                    />
                  </WithStyles(ListItem)>,
                  <WithStyles(ListItem)>
                    <WithStyles(ListItemIcon)>
                      <pure(FiberManualRecordIcon) />
                    </WithStyles(ListItemIcon)>
                    <WithStyles(ListItemText)
                      className=" progest-name"
                      secondary="undefined"
                    />
                  </WithStyles(ListItem)>,
                ],
                "dense": false,
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": Array [
                      <WithStyles(ListItemIcon)>
                        <pure(FiberManualRecordIcon) />
                      </WithStyles(ListItemIcon)>,
                      <WithStyles(ListItemText)
                        className=" lh-name"
                        secondary="undefined"
                      />,
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "class",
                      "props": Object {
                        "children": <pure(FiberManualRecordIcon) />,
                      },
                      "ref": null,
                      "rendered": Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "class",
                        "props": Object {
                          "className": undefined,
                        },
                        "ref": null,
                        "rendered": null,
                        "type": [Function],
                      },
                      "type": [Function],
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "class",
                      "props": Object {
                        "className": " lh-name",
                        "secondary": "undefined",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": [Function],
                    },
                  ],
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": Array [
                      <WithStyles(ListItemIcon)>
                        <pure(FiberManualRecordIcon) />
                      </WithStyles(ListItemIcon)>,
                      <WithStyles(ListItemText)
                        className=" fsh-name"
                        secondary="undefined"
                      />,
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "class",
                      "props": Object {
                        "children": <pure(FiberManualRecordIcon) />,
                      },
                      "ref": null,
                      "rendered": Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "class",
                        "props": Object {
                          "className": undefined,
                        },
                        "ref": null,
                        "rendered": null,
                        "type": [Function],
                      },
                      "type": [Function],
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "class",
                      "props": Object {
                        "className": " fsh-name",
                        "secondary": "undefined",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": [Function],
                    },
                  ],
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": Array [
                      <WithStyles(ListItemIcon)>
                        <pure(FiberManualRecordIcon) />
                      </WithStyles(ListItemIcon)>,
                      <WithStyles(ListItemText)
                        className=" oestro-name"
                        secondary="undefined"
                      />,
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "class",
                      "props": Object {
                        "children": <pure(FiberManualRecordIcon) />,
                      },
                      "ref": null,
                      "rendered": Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "class",
                        "props": Object {
                          "className": undefined,
                        },
                        "ref": null,
                        "rendered": null,
                        "type": [Function],
                      },
                      "type": [Function],
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "class",
                      "props": Object {
                        "className": " oestro-name",
                        "secondary": "undefined",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": [Function],
                    },
                  ],
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": Array [
                      <WithStyles(ListItemIcon)>
                        <pure(FiberManualRecordIcon) />
                      </WithStyles(ListItemIcon)>,
                      <WithStyles(ListItemText)
                        className=" progest-name"
                        secondary="undefined"
                      />,
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "class",
                      "props": Object {
                        "children": <pure(FiberManualRecordIcon) />,
                      },
                      "ref": null,
                      "rendered": Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "class",
                        "props": Object {
                          "className": undefined,
                        },
                        "ref": null,
                        "rendered": null,
                        "type": [Function],
                      },
                      "type": [Function],
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "class",
                      "props": Object {
                        "className": " progest-name",
                        "secondary": "undefined",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": [Function],
                    },
                  ],
                  "type": [Function],
                },
              ],
              "type": [Function],
            },
            "type": "div",
          },
          "type": [Function],
        },
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
